### ISSUE TITLE
No activity since 5month

### ISSUE THREAD

**Issue by sam2x** [mcasimir/mobile-angular-ui#335]
There is no activity on the project since 5month, Is it abandoned ? @mcasimir ?

Another question, any plans to integrate cleanly angular-material widgets with mobileangularui ? Since their project is being very promising/active on the Layout/UI part, it could give something very interesting to have "native" integration of it (or is that just because they use materializecss.com?).

**Comment by cyberkatze** [comment]
+1

**Comment by gmork** [comment]
+1

**Comment by negesti** [comment]
+1

**Comment by RoboSparrow** [comment]
+1

**Comment by mcasimir** [comment]
Hi guys, i'm really sorry. Your concerns are right. I had personal matters
(the good kind of) that drained all my time. I failed to create a good
community and a base of collaborators to overcome this 'single point of
failure'.

I never meant to abandon this project, and I started to work on Maui again
two days ago fixing some issue on website.

My will is to enable a solid process capable to match market expectations.

Further steps are:
- Take over the fake npm package
- Close the master and Work with MR
- Revisit the build pipeline to make it fully handleable by a build service
- Use pure karma + phantomjs + jasmine for unit testing
- Contribution guidelines
- Test, bump, changelog, tag, publish from pipeline.
- Share a public kanban board to groom and track the development.

After this setup I'll take care of major issues in github and being
prepared for Ng2 + Bs4.

Il giorno mar 23 feb 2016, 07:22 Joerg Boeselt <notifications@github.com>
ha scritto:
